### PR TITLE
Feat: TMS title bar

### PR DESCRIPTION
**Pull request by himanshi-incedo**

### CODE REVIEW

**Review by himanshi-incedo — COMMENTED** [review]
`LGTM`

File: package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "@testing-library/user-event": "^13.5.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-helmet": "^6.1.0",
```
Comment (kritikasinglaincedo) [inline package.json:11]: why are we using react-helmet?

File: src/App.js [hunk 2]
```diff
@@ -1,40 +1,49 @@
...
-        <Link to="/signin">{<button type="button" class="btn btn-outline-primary">Signin</button>}</Link>
-        <Link to="/signup">{<button type="button" class="btn btn-outline-primary">Signup</button>}</Link> */}
-
-        {/* <Route exact path="/" component={Signin} /> */}
-        {/* <Route path="/signin" component={Dashboard} /> */}
-        {/* <Route path="/signup" component={Signup} /> */}
-
-      </Router>
+      {/* <Router>
+        <Link to="/">
```
Comment (kritikasinglaincedo) [inline src/App.js:11]: Please remove the commented code

File: src/components/Signin.css [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  max-height: 99%;
+}
+
+/* Create two equal columns that sits next to each other */
+.column {
+  flex: 50%;
+  width: 50%;
+  padding: 10px;
+  height: 700px;
+  /* Should be removed. Only for demonstration */
```
Comment (kritikasinglaincedo) [inline src/components/Signin.css:12]: ??

File: src/components/Signin.css [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  display: flex;
+  align-content: center;
+  float: left;
+}
+
+.two {
+  background-color: #ff6620;
+  /* background-image: linear-gradient(315deg, #1CA7EC 0%, #3242ae 50%); */
```
Comment (kritikasinglaincedo) [inline src/components/Signin.css:20]: remove the commented code

**Review by kritikasinglaincedo — CHANGES_REQUESTED** [review]

**Pull request closed without merging**